src/main/resources/yaml/sample.yaml

```diff
@@ -0,0 +1,10 @@
+search:
+ check:
+  url: https://www.mcloud.de/web/guest/suche/-/results/searchAction?_mysearchportlet_query=
+  terms:
+   - autobahn
+   - bahn
+  
+ search-result:
+  xpath: blablabla
+  action: scrape
```

Comment:
If this file is made for testing or something similar, it shouldn't be in the `main` directory.